pyproject.toml

```diff
@@ -74,12 +74,14 @@ pytest = ">=7.2.2"
 pytest-cov = "^4.0.0"
 pytest-env = ">=0.8.1"
 pytest-mock = ">=3.10.0"
+# TODO: https://github.com/qiboteam/qibolab/pull/1128#issuecomment-2593208150
+dash = "^2.10.0"
```

Comment:
Since we added this, isn't it safe to remove the setuptools dependency above?

Reply:
Yes, it should. I'm about doing it